pygeoapi/provider/mvt_tippecanoe.py

```diff
@@ -169,22 +174,89 @@ def get_tiles(self, layer=None, tileset=None,
             except FileNotFoundError as err:
                 raise ProviderTileNotFoundError(err)
 
-    def get_metadata(self, dataset, server_url, layer=None,
-                     tileset=None, metadata_format=None, title=None,
-                     description=None, keywords=None, **kwargs):
+    def get_html_metadata(self, dataset, server_url, layer, tileset,
```

Comment:
Can we continue to render in `html` metadata fields for tippecanoe?

Reply:
Done! Thanks for the review, Francesco.